README.md

```diff
@@ -394,20 +394,43 @@ In Jetson nano, from January 22, 2021, it became possible to save the u-boot to
 
 ## How to use on an i.MX 8M Plus Phytec phyBOARD-Pollux board
 
+### Running from SD Card (phyBOARD-Pollux)
+
 1. Set bootmode switch (S3) to boot device from an SD card (positions 1,2,3,4 set to ON,OFF,OFF,OFF).
 2. Build a live image `make ZARCH=arm64 HV=kvm PLATFORM=imx8mp_pollux live-raw` (Only KVM is supported)
 3. Flash the `dist/arm64/current/live.raw` live EVE image onto your SD card by [following these instructions](#how-to-write-eve-image-and-installer-onto-an-sd-card-or-an-installer-medium)
 
+### Installing on eMMC (phyBOARD-Pollux)
+
+1. Build an installation raw image `make ZARCH=arm64 HV=kvm PLATFORM=imx8mp_pollux installer-raw` (Only KVM is supported)
+2. Flash the `dist/arm64/current/installer.raw` install EVE image onto your SD card by [following these instructions](#how-to-write-eve-image-and-installer-onto-an-sd-card-or-an-installer-medium)
+3. Set bootmode switch (S3) to boot device from an SD card (positions 1,2,3,4 set to ON,OFF,OFF,OFF).
```

Comment:
Is this step not needed when running the live image above?

Reply:
Might make sense to explain what they do (I speculate it is to allow writing to the eMMC but I have no idea)

Reply:
@eriknordmark , bootmode switches just selects from which device (SD Card or eMMC) the bootloader will be loaded by the ROM firmware. In a nutshell it selects from which device the board should boot. In order to install EVE on eMMC, we need first to boot the installer image from SD Card, that's why this step is needed. If you look to the instructions to run EVE from SD Card (on both i.MX8MP devices) this step is already there, so I think there is nothing to add here...

Reply:
OK